packages/js/product-editor/src/components/variations-table/variation-actions-menus/types.ts

```diff
@@ -7,15 +7,21 @@ import { MenuItem } from '@wordpress/components';
 export type VariationActionsMenuProps = {
 	disabled?: boolean;
 	selection: ProductVariation[];
-	onChange( values: PartialProductVariation[], showSuccess?: boolean ): void;
+	onChange(
+		values: PartialProductVariation[] | React.FormEvent,
+		showSuccess?: boolean
+	): void;
 	onDelete( values: PartialProductVariation[] ): void;
 };
 
 export type VariationQuickUpdateSlotProps = {
 	group: string;
 	supportsMultipleSelection: boolean;
 	selection: ProductVariation[];
-	onChange( values: PartialProductVariation[], showSuccess?: boolean ): void;
+	onChange(
+		values: PartialProductVariation[] | React.FormEvent< HTMLDivElement >,
```

Comment:
Is there a reason you decided to use the `HTMLDivElement` as part of the `FormEvent` here? and not in the `onChange` method above for the `VariationActionsMenuProps`?

Maybe also just stick with `React.FormEvent` here?

Reply:
It looks like that it was not necessary. I restored it b05df4440d9bc65fca97febed385210e92857b5a